charts/dex/templates/clusterrole.yaml

```diff
@@ -9,7 +9,44 @@ rules:
 - apiGroups: ["dex.coreos.com"] # API group created by dex
   resources: ["*"]
   verbs: ["*"]
-- apiGroups: ["apiextensions.k8s.io"]
-  resources: ["customresourcedefinitions"]
+- apiGroups:
+    - ""
+    - apps
+    - extensions
+    - policy
+    - "rbac.authorization.k8s.io"
+    - "storage.k8s.io"
+    - "networking.k8s.io"
+  resources:
+    - configmaps
+    - daemonsets
+    - deployments
+    - endpoints
+    - ingress
+    - ingresses
+    - namespaces
+    - nodes
+    - pods
+    - persistentvolumes
+    - persistentvolumeclaims
+    - replicasets
+    - replicationcontrollers
+    - serviceaccounts
+    - services
+    - storageclasses
+    - clusterroles
+    - roles
+    - events
+  verbs:
+    - get
+    - watch
+    - list
+    - create
+    - patch
+    - update
```

Comment:
Wait what ??? Do we really want to have (almost) everything ??
If I well understand it's only stuff doable by the UI no ? So only listing some stuff + maybe some edition no ?

Reply:
Agree with Teddy, this doesn't make sense. This ClusterRole is for Dex, so that it can use K8s API as its storage, it should have write permissions on such generic resources. Where does this come from? The `clusterrole` from upstream looks better: https://github.com/helm/charts/blob/master/stable/dex/templates/clusterrole.yaml

Reply:
yeah, i am going to keep the upstream one